src/main/java/org/graylog2/alarmcallbacks/pagerduty/PagerDutyClient.java

```diff
@@ -173,6 +175,19 @@ private PagerDutyEvent buildPagerDutyEvent(final Stream stream, final AlertCondi
         );
     }
 
+    protected String getAlarmBacklogSummary(List<Message> messages) {
+        final StringBuilder sb = new StringBuilder();
+        for (Message message : messages) {
+            final String msg = message.getMessage();
+            if (msg.length() > 225) {
+                sb.append("Message: {").append(msg.substring(0, 255)).append("... }  ");
```

Comment:
Shouldn't this use the same number in the condition (225) and inside the block (255)?